addons/im_livechat/i18n/im_livechat.pot

```diff
@@ -1221,6 +1241,14 @@ msgid ""
 "                visitors to talk in real time with your operators."
 msgstr ""
 
+#. module: im_livechat
+#. openerp-web
+#: code:addons/im_livechat/static/src/js/im_livechat.js:0
+#, python-format
+msgid ""
+"You took to long to send a message. Please refresh the page and try again."
```

Comment:
Here is the typo:
```suggestion
"You took too long to send a message. Please refresh the page and try again."
```

Reply:
good catch, your team is in charge of `im_livecaht` right?

Reply:
Yes that's why I checked that file in the first place (first file on my codeowners list).

Do we have a guide somewhere how to change a source term in a stable? I'm never sure how to avoid breaking stuff.

Reply:
yes, you don't :wink: you can change it in saas-13.whatever or master but if changed in stable, we lose the exisiting translations (as the message no longer match the source string)